revoke/job.go

```diff
@@ -0,0 +1,16 @@
+package revoke
+
+import (
+	"time"
+)
+
+type JobFunc func()
+
+func Schedule(interval time.Duration, job JobFunc) {
+	go func() {
+		for {
```

Comment:
if you want the job to happen in fixed intervals, no matter how long job() takes, use a Ticker:
```go
func job() {
    time.Sleep(500 * time.Millisecond)
}
func main() {
    for _ = range time.Tick(1 * time.Second) {
        fmt.Printf("NOW=%s\n", time.Now())
        job()
    }
}
```